.github/workflows/create-deployment.yaml

```diff
@@ -0,0 +1,63 @@
+---
+name: Create deployment
+
+on:
+  pull_request:
+    branches:
+      - master
+      - release/*
+    types:
+      - labeled
+env:
+  GITOPS_ENVIRONMENT: stage
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    if: contains(github.event.pull_request.labels.*.name, 'deploy_nodes')
```

Comment:
Maybe rename `deploy_nodes` -> `deploy_staging`? So that there could be different networks... each prefixed with `deploy_`?

Reply:
That should be agnostic to the user. It will be got from the target branch as per these comments https://github.com/hoprnet/hoprnet/pull/4761/files#diff-97e1e79b4ecbd5261fd1b1a49312400b613bd1953d84d3485019f8eb0667d2e6R164